docs/configexamples/ansible.rst

```diff
@@ -0,0 +1,302 @@
+	vyos8                      : ok=1    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+	vyos9                      : ok=1    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+
+Check the result on the vyos10 router:
+======================================
+
+.. code-block:: none
+
+	vyos@vyos10:~$ show interfaces
+	Codes: S - State, L - Link, u - Up, D - Down, A - Admin Down
+	Interface        IP Address                        S/L  Description
+	---------        ----------                        ---  -----------
+	eth0             192.0.2.108/24                  u/u  WAN
+	eth1             -                                 u/u  LAN
+	eth2             -                                 A/D
+	eth3             -                                 A/D
+	lo               127.0.0.1/8                       u/u
+					::1/128
+	
+	vyos@vyos10:~$ sh configuration commands | grep 8.8.8.8
+	set system name-server '8.8.8.8'
+
+The simple way without configuration of the hostname (one task for all routers):
+============================================================================
+
+.. code-block:: none
+
+	# nano /root/hosts_v2
+	[vyos_hosts]
+	vyos7 ansible_ssh_host=192.0.2.105
+	vyos8 ansible_ssh_host=192.0.2.106
+	vyos9 ansible_ssh_host=192.0.2.107
+	vyos10 ansible_ssh_host=192.0.2.108
+	[vyos_hosts:vars]
+	ansible_python_interpreter=/usr/bin/python3
+	ansible_user=vyos
+	ansible_ssh_pass=vyos
+	ansible_network_os=vyos
+	ansible_connection=network_cli
+
+	# nano /root/main_v2.yml
+	---
+	- hosts: vyos7,vyos8,vyos9,vyos10
+	
+	connection: network_cli
+	gather_facts: 'no'
+	
+	tasks:
+		- name: Configure remote
+		vyos_config:
+			lines:
+			- set system name-server 8.8.8.8
+			- set interfaces ethernet eth0 description WAN
+			- set interfaces ethernet eth1 description LAN
+			- set interfaces ethernet eth2 disable
+			- set interfaces ethernet eth3 disable
+			save:
+			true
+		  
+.. code-block:: none
+		  
+	# ansible-playbook -i hosts_v2 main_v2.yml
+	
+	PLAY [vyos7,vyos8,vyos9,vyos10] ************************************************
+	
+	TASK [Configure remote] ********************************************************
+	ok: [vyos8]
+	ok: [vyos7]
+	ok: [vyos9]
+	ok: [vyos10]
+	
+	PLAY RECAP *********************************************************************
+	vyos10                     : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+	vyos7                      : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+	vyos8                      : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+	vyos9                      : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+	
+
+In the next example, we'll use the Ansible with jinja2 templates and variables.
```

Comment:
I don’t  see Jinja2 files

Reply:
I'm going to show the example in the next PR.